sync/awsdatabricks.py

```diff
@@ -739,7 +743,7 @@ def _s3_contents_have_all_rollover_logs(contents: list[dict], run_end_time_secon
         # can proceed with the upload + prediction. If this proves to be finicky in some way, we can always
         # wait for the full 5 minutes after the run_end_time_seconds
         final_rollover_log
-        and final_rollover_log["LastModified"].timestamp() >= run_end_time_seconds
+        and final_rollover_log["LastModified"].timestamp() >= (run_end_time_seconds - 15)
```

Comment:
@gseyffert-sync Curious if you think this is ok. In the case of the jobs I tested on the last eventlog modification was consistently ~10 sec before the run end time.

Reply:
was the prior condition causing you to not be able to upload event logs? In general, I'd prefer to code against documented behaviour (as in the link in this comment) vs. observed behaviour

Reply:
do Job Bank runs not terminated the cluster at the end of the run? Because event logs are also purged when the cluster is terminated. In my manual tests, I didn't need to wait 5 minutes - only need to poll once or twice after the job finished (so, 15 - 30 seconds)

Reply:
Let's at least log when we fail to meet this condition before exhausting attempts. A warning sounds appropriate.

Reply:
We do log each time we poll (below on line 780), but it's an `info` log right now.

Reply:
Just reverted this change.

Job Bank doesn't explicitly terminate the cluster because it uses the `new_cluster` field of the JobCreate payload. The cluster terminates after the run, it just so happens in the case I looked at that the eventlog is last updated in s3 a little bit before the cluster terminates.